README.md

```diff
@@ -84,6 +84,12 @@ python manage.py dumpdata auth.user --indent 2
 
 While we're in the early stages of development it makes sense to delete existing migrations and create them afresh. To enable this, use the `make clear-db` command. Seed data will be re-applied when the application is restarted.
 
+### Changing the admin site theme
+
+The admin site is themed using the [django-admin-interface](https://pypi.org/project/django-admin-interface/) package. This is a temporary measure pending implementation of a proper design for the team-facing interface. The CSS values for the theme are loaded from JSON when the application is first brought up, either locally using the `init-local.sh` script or within one of the CodePipeline steps when deploying to AWS.
+
+The `django-admin-interface` package, in turn, loads these CSS values into admin site page as inline styles. Inline styles are, in general, forbidden by our content security policy so it is necessary to add a hash for the inline stylesheet to the `CSP_STYLE_SRC` variable in `settings.py`. If the styles are ever changed, the updated hash must be added as well.
+
```

Comment:
Could you add instructions (or a link) on how to generate the hashes? (and mention it's not just for admin pages but also for front-end inline styles and scripts)